APIs/schemas/edid.json

```diff
@@ -2,16 +2,27 @@
   "$schema": "http://json-schema.org/draft-07/schema#",
   "type": "object",
   "required": [
+    "version",
     "manufacturer",
     "manufacture_week",
     "manufacture_year",
-    "version",
+    "edid_binary_href",
```

Comment:
the challenge with "href" came with IS-10 about how to spec if it needed to be protected....

I think this could be done similar to IS-05 where the `/transportfile` endpoint returns the raw SDP file... Moving the `/raw_edid` endpoint you had to the sink would be sufficient

Reply:
I did it in 9056c6b.